test/scripts/integration_tests.py

```diff
@@ -181,34 +255,34 @@ def runTest(self):
         self.__check_metrics(6, 1, 3, 2, 1)
         self.__kubectl.delete(_NEMO_CUSTOM_RESOURCE_FILE, True)
         self.__check_metrics(7, 0, 3, 3, 1)
-        if self.__lostromos_process:
-            self.__lostromos_process.kill()
 
-        print("Starting Lostromos with helm controller")
+        self.__lostromos_process.kill()
         self.__lostromos_process = subprocess.Popen(
             [
                 _LOSTROMOS_EXE,
                 "start",
+                "--nop",
                 "--config",
-                _TEST_DATA_DIRECTORY + "/helm/wait-config.yaml",
-                "--helm-tiller",
-                self.__minikube_ip + ":32664",
-            ],
+                _LOSTROMOS_CONFIGURATION_FILE,
+                "--crd-filter",
+                "optIn",
```

Comment:
Looks like this needs to be changed to match the new filter and then the tests will pass.